types/result.go

```diff
@@ -249,6 +250,27 @@ func (r TxResponse) Empty() bool {
 	return r.TxHash == "" && r.Logs == nil
 }
 
+// SearchTxsResult defines a structure for querying txs pageable
+type SearchTxsResult struct {
+	TotalCount int          `json:"total_count"`
```

Comment:
all these `int` should instead be `uint64`

Reply:
The struct is for rest and tendermint return `int` too. So, maybe `int` is the right type.

```go
// github.com/tendermint/tendermint@v0.31.5/rpc/core/types/responses.go
// Result of searching for txs
type ResultTxSearch struct {
	Txs        []*ResultTx `json:"txs"`
	TotalCount int         `json:"total_count"`
}
```